### DISCUSSION TITLE
Ideas Examples for repository

### DISCUSSION THREAD

**Discussion by PedroSoaresDias** [vercel/next.js#63922] [Help]
### Summary

I'm unsure about having an idea of ​​how to contribute to the repository in the "examples" folder, I would like to know how I can talk about an idea in the Issues section or elsewhere on GitHub, I would like to contribute in some way?

**Comment by SouthLink** [comment]
https://github.com/vercel/next.js/blob/canary/contributing/examples/adding-examples.md

There might be something you want to know

**Reply by PedroSoaresDias** [reply]
When I was an idea, I go to in the Issues of repository speaking my idea or I can create my example project and make Pull Request?
I appreciate your help
